test/vm.py

```diff
@@ -80,6 +80,51 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, traceback):
         self.force_stop()
 
+    def execute_ansible(self, ansible_container_tag, playbook, username, password, env_vars=[], check=True):
```

Comment:
I wonder if we should move this out of VM into it's own testutil helper (or something). Right now the VM class is pretty targeted and has a thin profile. Adding this as a class method make it less so and AFAICT this helper can just use the VM and does not need to be part of it?